src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -193,9 +193,6 @@ public void testHeader(){
         Assert.assertEquals("r", schemaNames.get(1));
         Assert.assertEquals("a.age", schemaNames.get(2));
 
-        Assert.assertEquals(COLUMN_NODE, schemaTypes.get(0));
-        Assert.assertEquals(COLUMN_RELATION, schemaTypes.get(1));
-        Assert.assertEquals(COLUMN_SCALAR, schemaTypes.get(2));
```

Comment:
remove line 184
please annotate getSchemaTypes as `@deprecated` both in the interface and the implemented class, with explenation why it is deprecated